app/src/configs/env.ts

```diff
@@ -173,4 +173,4 @@ export const FHIR_API_BASE_URL = setEnv(
 
 export const KEYCLOAK_USERS_PAGE_SIZE = Number(setEnv('REACT_APP_KEYCLOAK_USERS_PAGE_SIZE', 20));
 
-export const USER_FORM_HIDDEN = setEnv('REACT_APP_USER_FORM_HIDDEN', '').split(',');
+export const USER_FORM_HIDDEN = setEnv('REACT_APP_USER_FORM_HIDDEN', 'contact').split(',');
```

Comment:
Should we maybe rethink this implementation? I feel like a single `ENABLE_CONTACT` env var with conditional rendering would be better?

Reply:
@machariamuguku This could definitely be improved, however, The form fields can be several, that would mean we would endup with several different env vars around the same thing.

Reply:
@p-netm fair enough. This probably deserves it's own PR but:
1. What if instead of looking for a disabled field, we look for an enabled one? This would make the default behaviour to not show unless set in the explicitly in the env. e.g `not_hidden?.includes("contact")? <Contact/>: null`. Such that if `REACT_APP_UNHIDE_USER_FIELD === ""` the contact field won't show. We'll avoid having to pass every field that's not rendered by default to the setEnv fn.
2. What if we hide the field with conditional rendering {render && <Rendered/>} instead of css (hidden=true). This will be clear from the onset that the field is rendered conditionally

Reply:
@p-netm is this part related to PR ? i am not sure what we are following over here or how does this changes the implementation

Reply:
> @p-netm fair enough. This probably deserves it's own PR but:
> 
>     1. What if instead of looking for a disabled field, we look for an enabled one? This would make the default behaviour to not show unless set in the explicitly in the env. e.g `not_hidden?.includes("contact")? <Contact/>: null`. Such that if `REACT_APP_UNHIDE_USER_FIELD === ""` the contact field won't show. We'll avoid having to pass every field that's not rendered by default to the setEnv fn.
> 
>     2. What if we hide the field with conditional rendering {render && } instead of css (hidden=true). This will be clear from the onset that the field is rendered conditionally

I like this, kind of alligns with staled ongoing discussions here: #168, yeah its definitely stomething we can enforce for all forms. so we will have disabled fields, hidden fields, renderedFields

Reply:
> etm is this part related to PR ? i am not sure what we are following over here or how does this changes the implementation

not quite sure i understand what you mean. but the env change is meant to avoid having to update existing web deployment instances so that the contact field does not show up there if not needed.

Reply:
@p-netm do we then have anything actionable? Should we create an issue?

Reply:
yeah, will rework this field in another pr, then create an issue to follow the convention for existing and new forms